test/pensions/containers/ConfirmationPage.unit.spec.jsx

```diff
@@ -10,7 +10,7 @@ const form = {
       attributes: {
         confirmationNumber: 'V-PEN-177',
         regionalOffice: [
-          'Western Region',
+          'Attention: Western Region',
```

Comment:
I assume this was the only difference between what we expected and what was returned, yeah?